src/test/java/com/google/javascript/gents/multiTests/converts_ts_module_require/converts_ts_module_require.ts

```diff
@@ -0,0 +1,4 @@
+import './already_converted_to_ts_keep';
```

Comment:
Does this automatically calculate the correct relative path? I was expecting you to just turn the namespace into a path, i.e. `google3/already/ts/module`